[2023-12-14T19:41:14.314+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: open_fda.transform manual__2023-12-14T19:40:42.602182+00:00 [queued]>
[2023-12-14T19:41:14.318+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: open_fda.transform manual__2023-12-14T19:40:42.602182+00:00 [queued]>
[2023-12-14T19:41:14.318+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 1
[2023-12-14T19:41:14.323+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): transform> on 2023-12-14 19:40:42.602182+00:00
[2023-12-14T19:41:14.327+0000] {standard_task_runner.py:57} INFO - Started process 72 to run task
[2023-12-14T19:41:14.330+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'open_fda', 'transform', 'manual__2023-12-14T19:40:42.602182+00:00', '--job-id', '4', '--raw', '--subdir', 'DAGS_FOLDER/outcome_dag.py', '--cfg-path', '/tmp/tmpx3w6gpsc']
[2023-12-14T19:41:14.331+0000] {standard_task_runner.py:85} INFO - Job 4: Subtask transform
[2023-12-14T19:41:14.365+0000] {task_command.py:416} INFO - Running <TaskInstance: open_fda.transform manual__2023-12-14T19:40:42.602182+00:00 [running]> on host 56fd4118eea6
[2023-12-14T19:41:14.417+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='open_fda' AIRFLOW_CTX_TASK_ID='transform' AIRFLOW_CTX_EXECUTION_DATE='2023-12-14T19:40:42.602182+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-14T19:40:42.602182+00:00'
[2023-12-14T19:41:15.308+0000] {logging_mixin.py:154} INFO - Tables have been created
[2023-12-14T19:41:15.402+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.8/site-packages/***/utils/context.py:314: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2023-12-14T19:41:17.689+0000] {warnings.py:109} WARNING - /opt/***/dags/etl/transformation.py:61: SettingWithCopyWarning: 
A value is trying to be set on a copy of a slice from a DataFrame.
Try using .loc[row_indexer,col_indexer] = value instead

See the caveats in the documentation: https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html#returning-a-view-versus-a-copy
  new_values[f'{col}_id'] = range(max_id + 1, max_id + 1 + len(new_values))

[2023-12-14T19:41:21.976+0000] {logging_mixin.py:154} INFO - File uploaded successfully to 'dsc-bucket-1/transformed/2023-12-14_recall_fct.csv'.
[2023-12-14T19:41:23.199+0000] {logging_mixin.py:154} INFO - File uploaded successfully to 'dsc-bucket-1/transformed/2023-12-14_firm_dim.csv'.
[2023-12-14T19:41:23.878+0000] {logging_mixin.py:154} INFO - File uploaded successfully to 'dsc-bucket-1/transformed/2023-12-14_status_dim.csv'.
[2023-12-14T19:41:24.571+0000] {logging_mixin.py:154} INFO - File uploaded successfully to 'dsc-bucket-1/transformed/2023-12-14_cause_dim.csv'.
[2023-12-14T19:41:24.578+0000] {python.py:194} INFO - Done. Returned value was: None
[2023-12-14T19:41:24.601+0000] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=open_fda, task_id=transform, execution_date=20231214T194042, start_date=20231214T194114, end_date=20231214T194124
[2023-12-14T19:41:24.651+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-14T19:41:24.667+0000] {taskinstance.py:2778} INFO - 1 downstream tasks scheduled from follow-on schedule check
